app/components/layout.js

```diff
@@ -41,6 +42,9 @@ class Layout extends React.Component {
     return (
       <div>
         <Navbar {...this.props} connect={this.connect} handleDisconnect={this.disconnect}/>
+        {
+          (this.props.status === 'FAILED') ? <Alert type="danger">{this.props.message}</Alert> : ''
```

Comment:
I am not finalised yet on what all constants need to be there.
Also need to identify what all constants for middleware and what all for
our app.
Once we done it we will move all to constants.

On Wed, Mar 23, 2016 at 4:12 PM, Yuva <notifications@github.com> wrote:

> In app/components/layout.js
> <https://github.com/code-mancers/rbkit-electron-redux/pull/9#discussion_r57138786>
> :
>
> > @@ -41,6 +42,9 @@ class Layout extends React.Component {
> >      return (
> >        <div>
> >          <Navbar {...this.props} connect={this.connect} handleDisconnect={this.disconnect}/>
> > +        {
> > +          (this.props.status === 'FAILED') ? <Alert type="danger">{this.props.message}</Alert> : ''
>
> maybe its a good idea to extract FAILED, CONNECTING strings into
> constants.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/code-mancers/rbkit-electron-redux/pull/9/files/b5077fdb7adf104551977f5c9565030567154b5e#r57138786>
>